### PR TITLE
fix: diff of Doctrine annotations

### PR DESCRIPTION
**Pull request by bouffa**
I had to change these lines to fix some issues :

Line 406

```diff
-#[ORM\Column(type="text")]
+#[ORM\Column(type: "text")]
```

Line 420 and Line 481

```diff
-#[ORM\Column]
+#[ORM\Column(type: "datetime")]
```

### CODE REVIEW

**Review by welcoMattic — APPROVED** [review]

File: distribution/index.md [hunk 1]
```diff
@@ -374,57 +374,57 @@ Modify these files as described in these patches:
...
 -    /** The id of this book. */
 +    /**
 +     * The id of this book.
 +     */
-     #[ORM\Id, ORM\Column, ORM\GeneratedValue]
++    #[ORM\Id, ORM\Column, ORM\GeneratedValue]
      private ?int $id = null;
  
 -    /** The ISBN of this book (or null if doesn't have one). */
 +    /**
```
Comment (alanpoulain) [inline distribution/index.md:389]: Why changing the PHPDoc?

Comment (welcoMattic) [reply distribution/index.md:389]: The PHPDoc is the only "diffed" thing in the actual page, see https://api-platform.com/docs/distribution/#plugging-the-persistence-system

Comment (alanpoulain) [reply distribution/index.md]: I was perturbed by GitHub color syntaxing, thanks!

**Comment by alanpoulain** [comment]
Thanks!

**Pull request merged**